src/cipher/encryptingstorage/README.txt

```diff
@@ -175,9 +175,10 @@ from an decrypted database to a encrypted one, as in::
     True
 
     >>> import ZODB.utils
+    >>> from __future__ import print_function
```

Comment:
The test here doesn't actually use print, except if there is an error.
But yes, just to be sure I tested it with a print:

```
    >>> import ZODB.utils
    >>> from __future__ import print_function
    >>> print('oops', i)
    oops 1999
```

without the print_function:

```
    >>> import ZODB.utils
    >>> print('oops', i)
    ('oops', 1999)
```